lib/internal/Magento/Framework/Crontab/Test/Unit/CrontabManagerTest.php

```diff
@@ -354,6 +376,17 @@ public function saveTasksDataProvider()
                     . ' %% cron:run | grep -v \"Ran \'jobs\' by schedule\"' . PHP_EOL
                     . CrontabManagerInterface::TASKS_BLOCK_END . ' ' . hash("sha256", BP) . PHP_EOL,
             ],
+            [
+                'tasks' => [
+                    ['command' => '{magentoRoot}run.php % cron:run | grep -v "Ran \'jobs\' by schedule"$']
```

Comment:
I think it would be better to make some more real case, for instance with `$(date)`